packages/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.1.3
+
+* Fix variouse bugs.
+* Update webview_flutter_tizen to 0.5.1.
+
+## 0.1.2
+
+* Update dependency package.
+
```

Comment:
Is the version related changes in progress? 

```suggestion
## 0.1.2

* Fix variouse bugs.
* Update webview_flutter to 3.0.4.
* Update webview_flutter_tizen to 0.5.1.

```

And all version related changes in `pubspec.yaml`s and `README`s.

Reply:
Oh, I am sorry for being too late. I fixed them. Thank you 😀